packages/opentelemetry-core/src/context/propagation/composite.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Carrier, Context, HttpTextFormat } from '@opentelemetry/api';
+
+/** Combines multiple propagators into a single propagator. */
+export class CompositePropagator implements HttpTextFormat {
+    private _propagators: HttpTextFormat[];
+    constructor(...propagators: HttpTextFormat[]) {
```

Comment:
I think we should add a comment here about the order of the `propagators`. If the ordering is not right, that might result in some performance penalty because we are going through all provided propagators one-by-one.

Reply:
All propagators are run no matter what. It is never short circuited. I will still add the comment though because if two propagators write to the same key, the first will be overwritten by the second.